crates/node_binding/binding.d.ts

```diff
@@ -862,10 +857,3 @@ export interface RawTrustedTypes {
   policyName?: string
 }
 
-export interface ThreadsafeNodeFS {
-  writeFile: (...args: any[]) => any
-  mkdir: (...args: any[]) => any
-  mkdirp: (...args: any[]) => any
-  removeDirAll: (...args: any[]) => any
-}
```

Comment:
Where's this from?

Reply:
I don't know, maybe it is a bug of napi

Reply:
it's a known bug of napi, I'm not sure we have any issue for this @h-a-n-a @IWANABETHATGUY